lib/nationsync/cli.rb

```diff
@@ -50,9 +53,10 @@ def init
       "session_id" => auth.session_id
     }
     save_config!
-    puts "Created: #{fn}"
+    puts "Created .nbconfig1\n"
+    puts "Now run \"nationsync pick_theme\" to pick the theme you want to sync"
```

Comment:
nit. Can use single quotes on the outside so that you don't have to escape the inner double-quotes:

```ruby
puts 'Now run "nationsync pick_theme" to pick the theme you want to sync'
```